infra/terraform/modules/data_access/membership_events/membership_events.py

```diff
@@ -39,10 +34,15 @@ def event_received(sns_event, context):
         if event["scope"] != "team":
             continue
 
-        if action in EVENT_HANDLERS:
+        if action == "added":
             invoke_lambda(
-                function=EVENT_HANDLERS[action],
-                payload=payload(event),
+                function=os.environ["LAMBDA_ATTACH_BUCKET_POLICY_ARN"],
+                payload=attach_payload(event),
+            )
+        elif action == "removed":
+            invoke_lambda(
+                function=os.environ["LAMBDA_DETACH_BUCKET_POLICIES_ARN"],
+                payload=detach_payload(event),
```

Comment:
I'm looking at the code to see how to remove this duplication. I may come to your desk to discuss this face to face.